### PR TITLE
code updates for one step cold start

### PR DESCRIPTION
**Pull request by junwang-noaa**
### Description of changes

This PR contains the code changes for one step cold start. 

### Specific notes

Issues Fixed (include github issue #):

Are changes expected to change answers?
- bit for bit (yes) with current run sequence

Specific changes:
- changes in parm directory (yes)
- changes in module files (no)
- new tests added (yes)

Testing performed:
- machines: hera, orion, dell
- details: 

Hashes used for testing:
- NEMS: 9d05172b
- CMEPS: 5e63bd4
- FV3: 90d5eaff
- MOM6: e42147c5
- CICE: 285985c
- WW3: 96e3f3a8
- FMS: f61416fe
- stochastic_physics: ffdd19bc

Co-authored by: Denise Worthen <Denise.Worthen@noaa.gov>

Dependency:

[FV3 PR #168](https://github.com/NOAA-EMC/fv3atm/pull/168)
[MOM6 PR #33](https://github.com/NOAA-EMC/MOM6/pull/33)

[s2s PR#176](https://github.com/ufs-community/ufs-s2s-model/pull/176)

### CODE REVIEW

**Comment by DeniseWorthen** [comment]
I don't think we need a special nems.configure.medcoldcmeps_atm_ocn_ice_1step.IN. 

The test (cpld_fv3_ccpp_mom6_cice_cmeps_1stepcold_atm_flux) can use the existing nems.configure.medcmeps_atm_ocn_ice.IN but in the test you would set the variable use_coldstart = false:

In the test: 

export NEMS_CONFIGURE="nems.configure.medcmeps_atm_ocn_ice.IN"
export use_coldstart="false"

**Comment by junwang-noaa** [comment]
I was testing it. So far this is the difference in the two settings that we
need to consolidate:

restart_n,  using @[FV3_RESTART_INTERVAL] default value 0 causes issues in
ICE tasks
<       case_name = ufs.s2s
<       restart_n = @[FV3_RESTART_INTERVAL]
---
>       case_name = ufs.s2s.cold
>       restart_n = @[FHMAX]

I can change  FV3_RESTART_INTERVAL to be a non-zero value at this time.

**Comment by DeniseWorthen** [comment]
That's right. As long as FV3 considers restart_interval=0 to be equivalent to restart_interval=fhmax we should be fine. 

The case name determines what string the mediator restart file name will start with. For the 1-step coldstart, we want it to write a mediator file with the case_name "ufs.s2s".

File: tests/tests/cpld_fv3_ccpp_mom6_cice_cmeps_1stepcold_atm_flux [hunk 1]
```diff
@@ -0,0 +1,91 @@
+#
+#  CPLD_FV3_CCPP_MOM6_CICE_CMEPS_1STEPCOLD_ATM_FLUX test
+#
+
+export TEST_DESCR="Fully coupled FV3-CCPP-MOM6-CICE-CMEPS system - 1 step cold start"
+
+export CNTL_DIR="RT-Baselines_1stepcold_ccpp_cmeps"
+export CNTLMED_DIR="RT-Baselines_1stepcold_ccpp_cmeps"
```
Comment (DeniseWorthen) [inline tests/tests/cpld_fv3_ccpp_mom6_cice_cmeps_1stepcold_atm_flux:8]: I think can also be an empty string since there is no control mediator directory.

**Review by DeniseWorthen — APPROVED** [review]
Looks fine.

**Review by binli2337 — APPROVED** [review]

**Comment by junwang-noaa** [comment]
Denise/Bin, this PR is ready to merge.

**Pull request merged**